src/smsCommandFields/DataElementTimesCategoryOptionCombosCompletenessMessage.js

```diff
@@ -0,0 +1,47 @@
+import { NoticeBox, ReactFinalForm } from '@dhis2/ui'
+import React from 'react'
+
+import {
+    ALL_DATAVALUE,
+    AT_LEAST_ONE_DATAVALUE,
+    FIELD_COMMAND_COMPLETENESS_METHOD_NAME,
+} from '../smsCommandFields'
+import { FormRow } from '../forms'
+import i18n from '../locales'
+
+const { useField } = ReactFinalForm
+
+export const DataElementTimesCategoryOptionCombosCompletenessMessage = () => {
```

Comment:
This name is too long in my opinion. There's a couple of things we should address later in this PR in my opinion, the name here would be one of them if you ask me. So not a blocker, but I couldn't not comment on it 😅